book/03-git-branching/sections/rebasing.asc

```diff
@@ -55,6 +55,8 @@ Wenn man das Protokoll eines rebasierten Branches betrachtet, sieht es aus wie e
 
 Häufig werden Sie das anwenden, damit Ihre Commits sauber auf einen Remote-Zweig angewendet werden – vielleicht in einem Projekt, zu dem Sie beitragen möchten, das Sie aber nicht pflegen.
 In diesem Fall würden Sie Ihre Arbeiten in einem Branch erledigen und im Anschluss Ihre Änderungen mittels Rebase zu `origin/master` hinzufügen, wenn Sie soweit sind, Ihre Patches dem Hauptprojekt zu übermitteln.
+// Der letzte Halbsatz klingt (auch im englischen Original) komisch.
+// Ich weiß auch in beiden Sprachen nicht, was gemeint sein könnte.
```

Comment:
Lies dazu bitte in den [Manpages git-merge](https://git-scm.com/docs/git-merge) (Options --commit und --cleanup=<mode>) nach.
Ich glaube, das ist hier gemeint.

Reply:
Nicht jetzt, dazu ist die Seite zu lang und ich zu müde. Im Englischen ist der Satz

> That way, the maintainer doesn't have to do any integration work -- just a fast-forward or a clean apply.

Meines Erachtens fehlt im letzten Halbsatz etwas. Ich nehme an, dass mit "fast-forward" ein "fast-forward merge" gemeint ist. Aber was soll denn "a(n) apply" sein? "Apply" gibt es als Nomen (im Englischen m.E. nicht.

Dieser Satz ist also sehr wahrscheinlich eine Abkürzung. Stellt sich dem Nicht-Git-Kenner also die Frage: Wofür? Könntest du den Satz vielleicht mal in "lang" formulieren?

Reply:
Lies bitte die paar Zeilen bei dem Abschnitt "Options" --> "--commit" und "--cleanup"
Dort ist, glaube ich, das etwas genauer erklärt.

Reply:
Ich verstehe es trotzdem nicht ... (Muss ich aber ja im Zweifel auch nicht. Ist ja möglich, dass nur _ich_ zu blöd bin.)
Wenn du aber eine lange Version formulieren kannst ...

Keine Eile. Ist ja alles ein Freizeitprojekt.